### DISCUSSION TITLE
Bulk actions (mark as reviewed) or import as reviewed

### DISCUSSION THREAD

**Discussion by cdes** [tolgee/tolgee-platform#1439] [Ideas]
Hello,

It would be great to have such a feature, because it's really tedious to click reviewed 5000 times.

Valid use cases:

1. Creating a backup and importing in another project, all imported strings are "not reviewed".
2. Moving from another tool to Tolgee, (this affected me and I've had to sit for 2 hours clicking reviewed more than 5000 times).
3. Importing an older file.

As you can see, this is tightly coupled and affected by the import feature.
This means that providing a "mark as reviewed" while importing might be enough.

What do you think?

**Comment by JanCizmar** [comment]
Hey! You're right. This is not optimal. We plan to implement it in the next few months. We need to implement a unified way to handle batch operations even when they're time-consuming. We plan to run the operations as asynchronous tasks and inform the user about the progress using WebSockets. This will take some time to implement. 

Here is the corresponding issue: https://github.com/tolgee/tolgee-platform/issues/911

**Reply by cdes** [reply]
That's awesome!  I wish I knew Kotlin, I would've helped.
